std_msgs/Header header

int64 node

geometry_msgs/Transform laser_transform
geometry_msgs/Transform zed0_transform
geometry_msgs/Transform zed1_transform
geometry_msgs/Transform zed2_transform

sensor_msgs/PointCloud2 cloud

sensor_msgs/Image zed0_image
sensor_msgs/CameraInfo zed0_cinfo

sensor_msgs/Image zed1_image
sensor_msgs/CameraInfo zed1_cinfo

sensor_msgs/Image zed2_image
sensor_msgs/CameraInfo zed2_cinfo
